Samples/auth/Outlook-Event-SSO-NAA/README.md

```diff
@@ -0,0 +1,154 @@
+---
+page_type: sample
+urlFragment: outlook-event-sso-naa
+products:
+  - office
+  - office-outlook
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: "08/20/2024 10:00:00 AM"
+description: "This sample shows how to implement SSO in an event in an Outlook add-in by using nested app authentication."
+---
+
+# Use SSO in events in an Outlook add-in using nested app authentication (preview)
```

Comment:
```suggestion
# Implement SSO in events in an Outlook add-in using nested app authentication (preview)
```
Just trying to avoid use and using in the same sentence. I suggest swapping the verb with the first sentence in the first paragraph. This won't affect SEO.